### ISSUE TITLE
Unable to define Harvest moon Ritual Area

### ISSUE THREAD

**Issue by afghan2007** [WayofTime/BloodMagic#984]
#### Steps to reproduce:

1. Create the ritual of Harvest Moon and a ritual tinkerer
2. Attempt to redefine the harvest moons area of effect
3. If you can either get a success or fail message that the ritual tinkerer has actually done its job then you have Not duplicated my issue, and I am just going crazy. PS, I have been able to redefine other rituals with no issue.
...

____
#### Affected Versions (Do *not* use "latest"):

- BloodMagic: BloodMagic-1.9.4-2.1.0-66
- Minecraft: 10.2
- Forge: 1.10.2-12.18.2.2114

**Comment by WayofTime** [comment]
Is the ritual activated? The ritual bounds will also be fixed in the next version.

**Comment by afghan2007** [comment]
Ive been trying to test the ritual in creation to see if it is working and determine its default range.  I have tried activating it repeatedly.  To date the ritual has been placed one block above and 2 blocks above a fielded of crop and it has not done anything to the fully mature wheat.  I think the ritual may be broken?  Still unable to create a defined area. 

Ill probably put it of as my last automation attempt for now.  Don't work too hard today,  Happy Thankgiving.

**Comment by afghan2007** [comment]
Hmm, I was able to set a vertical range,  after that occurred I suddenly seemed to acquired the ability to set horizontal ranges...I'm sorry to waste your time, the issue seems to be on my end.

Confirmed that I must define a vertical limit first before I can set a horizontal area.  I created a new ritual with the same issues and following this step fixed it.  Still haven't been successful observing the normal reach of the ritual.
